pom.xml

```diff
@@ -11,7 +11,7 @@
 
     <groupId>com.elasticbox.jenkins-ci.plugins</groupId>
     <artifactId>elasticbox</artifactId>
-    <version>5.0.0-SNAPSHOT</version>
+    <version>5.0.1</version>
```

Comment:
Please, keep 5.0.0-SNAPSHOT, the plugin release process (not yet done) will take care of this. (@afavieres, FYI)